.github/workflows/generatedoxygen.yml

```diff
@@ -1,9 +1,9 @@
-name: generatedoxygen
+name: Generate Doxygen
 
 on:
   push:	
     branches:	
-      - master
+      - makefile
```

Comment:
hat ponthogy a master-en kellene lemennie, nem? Amikor oda bemegy valami, az deploy-olodjon le. A gh-pages-t nem ertem, ott nincs ott a kod, nincs mire lefutnia ennek.

Reply:
makefile-t eredetileg azért raktam bele, hogy meggyőződjek róla, hogy lefut a GHA, csak bent maradt, a gh-pages meg benne volt eddig is, nem tudom miért 😅